### PR TITLE
Add myself to milestone-maintainers

### PR DESCRIPTION
**Pull request by feiskyer**
**What this PR does / why we need it**:

Add myself to milestone-maintainers and sort the list alphabetically. I'm already in kubernetes/kubernetes-milestone-maintainers team.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -115,78 +115,79 @@ aliases:
...
-    - abgworrall
-    - krzyzacy
     - steveperry-53
-    - radhikpac
-    - jpbetz
-    - cmluciano
-    - bsalamat
-    - m1093782566
+    - thockin
+    - timothysc
```
Comment (k82cn) [inline OWNERS_ALIASES]: duplicated.

**Comment by thockin** [comment]
can you not sort it for now, and do that as a followup or as a second commit?

**Comment by feiskyer** [comment]
@thockin @k82cn Split into two commits and removed duplications. PTAL.

**Comment by feiskyer** [comment]
ping @thockin

**Comment by feiskyer** [comment]
@brendandburns Removed second commit. PTAL

**Comment by feiskyer** [comment]
ping @brendandburns @thockin

**Comment by feiskyer** [comment]
@brendandburns @thockin Still not get approved. Could you help to approve this?

**Pull request merged**